src/vs/base/browser/ui/scrollbar/scrollableElement.ts

```diff
@@ -240,12 +240,12 @@ export class ScrollableElement extends Widget {
 			let deltaX = e.deltaX * this._options.mouseWheelScrollSensitivity;
 
 			if (this._options.flipAxes) {
-				deltaY = e.deltaX;
-				deltaX = e.deltaY;
+				deltaY = deltaX;
```

Comment:
Sorry if I'm missing something, but how does this work?

```
deltaY = deltaX;
deltaX = deltaY;
```

will not actually swap the two variables, it will just assign them both to have the value of deltaX. Is that the desired behavior?